qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -106,6 +129,7 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
     }
 
     @Tag("Integration")
+    @Tag("Parallel")
```

Comment:
I'm unfamiliar with `@Tag("Parallel")` - does it mean all tests with this tag will run concurrently?

Reply:
Yes. There are three gradle tasks run for groovy tests. The middle one `testParallel` (or `testParallelBAT` for PRs) will run all the specifications with `@Tag("Parallel")` concurrently. Note, its the specifications (each groovy class file) and not the features (def "test me") that run concurrently.